app/Http/Controllers/TeamController.php

```diff
@@ -113,18 +113,14 @@ public function destroy(Team $team) {
 	 * @return \App\Course
 	 */
 	private function createTeam(array $attributes, $image) {
+		$path = isset($image) ? Storage::putFile('teams/avatars', $image) : null;
```

Comment:
Sorry, algo que se me pasó decir es que nunca quieres guardar archivos con el nombre con el que te los manda el usuario (it's a security concern). Aquí puedes generar un nombre único y pusar `putFileAs` con algo como:

```PHP
md5(date("Ymdhis",time()).$image['file_name'].rand(0,1000))
```
(Not bullet-proof, but works for our purposes)

Solo que va a haber que separar la extensión manualmente o vía el type del archivo.

Reply:
Cool! So long as you're sure :100: